packages/web/app/src/components/organization/Permissions.tsx

```diff
@@ -150,19 +150,17 @@ export const PermissionScopeItem = <
                   : false;
 
               return (
-                <>
```

Comment:
During local testing I found that this extra wrapper caused a warning because the list of permission scope items needs the root element to have a unique key, but the key attribute is set on the SelectItem.
Removing this wrapper lets the key be on the root element and resolves this.